main/queues.h

```diff
@@ -0,0 +1,4 @@
+#include "freertos/queue.h"
+
+static QueueHandle_t tx_task_queue;
```

Comment:
do these need to be static?

Reply:
outside of a function context, the static keyword in c is mostly used as a translation unit visibility specifier like private, so I think we leave them as static to avoid leaking the `tx_task_queue` name since it seems like a pretty common name.